base/v0_5_exp/schema.go

```diff
@@ -210,6 +211,7 @@ type Resource struct {
 }
 
 type SSHAuthorizedKey string
+type SSHAuthorizedKeyFile string
```

Comment:
We define some type aliases for historical reasons, but I'd be inclined to skip this.  By comparison, `Resource.Local` is `*string`.